src/ucs/debug/debug.c

```diff
@@ -1163,11 +1163,12 @@ int sigaction(int signum, const struct sigaction *act, struct sigaction *oact)
     return orig_sigaction(signum, act, oact);
 }
 
+extern ucs_profile_global_context_t ucs_profile_global_ctx;
```

Comment:
remove, include profile.h

Reply:
if it's used even when ucx is built without profile (where?) then need to move it to profile.h instead of profile_on.h

Reply:
ok I will move it. It's used to dump profile. If ucx is built without profile but application explicitly include profile_on.h for example in test/apps/profiling/ucx_profiling test